ts/screens/profile/ProfileMainScreen.tsx

```diff
@@ -299,16 +302,125 @@ class ProfileMainScreen extends React.PureComponent<Props, State> {
+      isPagoPATestEnabled,
+      navigation,
+      notificationId,
+      notificationToken,
       sessionToken,
       walletToken,
-      notificationToken,
-      notificationId
+      setDebugModeEnabled
     } = this.props;
+    const { deviceUniqueId } = this.state;
+
+    return (
+      <React.Fragment>
+        <SectionHeaderComponent
+          sectionHeader={I18n.t("profile.main.developersSectionHeader")}
+        />
+        {isPlaygroundsEnabled && (
+          <>
+            <ListItemComponent
+              title={"MyPortal Web Playground"}
+              onPress={() => navigation.navigate(ROUTES.WEB_PLAYGROUND)}
+            />
+            <ListItemComponent
+              title={"Markdown Playground"}
+              onPress={() => navigation.navigate(ROUTES.MARKDOWN_PLAYGROUND)}
+            />
+          </>
+        )}
+
+        {/* Showroom */}
+        <ListItemComponent
+          title={I18n.t("profile.main.showroom")}
+          onPress={() => navigation.navigate(ROUTES.SHOWROOM)}
+          isFirstItem={true}
+        />
+
+        {this.developerListItem(
+          I18n.t("profile.main.pagoPaEnvironment.pagoPaEnv"),
+          isPagoPATestEnabled,
+          this.onPagoPAEnvironmentToggle,
+          I18n.t("profile.main.pagoPaEnvironment.pagoPAEnvAlert")
+        )}
+        {this.developerListItem(
+          I18n.t("profile.main.debugMode"),
+          isDebugModeEnabled,
+          setDebugModeEnabled
+        )}
+        {isDebugModeEnabled && (
+          <React.Fragment>
+            {backendInfo &&
+              this.debugListItem(
+                `${I18n.t("profile.main.backendVersion")} ${
+                  backendInfo.version
+                }`,
+                () => clipboardSetStringWithFeedback(backendInfo.version),
+                false
+              )}
+
+            {isDevEnv &&
+              sessionToken &&
+              this.debugListItem(
+                `Session Token ${sessionToken}`,
+                () => clipboardSetStringWithFeedback(sessionToken),
+                false
+              )}
+
+            {isDevEnv &&
+              walletToken &&
+              this.debugListItem(
+                `Wallet token ${walletToken}`,
+                () => clipboardSetStringWithFeedback(walletToken),
+                false
+              )}
+
+            {isDevEnv &&
+              this.debugListItem(
+                `Notification ID ${notificationId.slice(0, 6)}`,
+                () => clipboardSetStringWithFeedback(notificationId),
+                false
+              )}
+
+            {isDevEnv &&
+              notificationToken &&
+              this.debugListItem(
+                `Notification token ${notificationToken.slice(0, 6)}`,
+                () => clipboardSetStringWithFeedback(notificationToken),
+                false
+              )}
+
+            {isDevEnv &&
+              this.debugListItem(
+                `Device unique ID ${deviceUniqueId}`,
```

Comment:
Usually this string is long and complicated. Could it make senso to use the `splice(0,6)` used for others items?

Reply:
The style is already taking care of longer strings via ellipsis based on the view size: I believe it is better than an arbitrary trim tbh.

Reply:
At this point what to you think to make the avoid splicing the others values?
So we handling all this token in the same way

Reply:
Updated in https://github.com/pagopa/io-app/pull/3159/commits/d94e1d354bb625816678e8304d8ca4849f565946

This is a screenshot on Android:

![Screenshot 2021-06-22 at 10 25 38](https://user-images.githubusercontent.com/2094604/122890964-6fb06480-d344-11eb-93b5-d2c0a9562915.png)